app/core/EngineService/EngineService.ts

```diff
@@ -54,15 +54,19 @@ class EngineService {
         key: `${engine.context.AddressBookController.name}:stateChange`,
       },
       { name: 'AssetsContractController' },
-      { name: 'NftController' },
+      { name: 'NftController', key: 'NftController:stateChange' },
       {
         name: 'TokensController',
+        key: 'TokensController:stateChange',
```

Comment:
We could keep the same pattern
```suggestion
        key: `${engine.context.TokensController.name}:stateChange`,,
```